packages/sui-test/bin/sui-test-browser.js

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+/* eslint no-console:0 */
+
+require('./karma/patch')
+const program = require('commander')
+const chalk = require('chalk')
+const runner = require('./karma')
+const { cleanStack } = require('./karma/util')
+
+program
+  .option('-W, --watch', 'Run in watch mode')
+  .option('-C, --ci', 'Run a Firefox headless for CI testing')
+  .on('--help', () => {
+    console.log('  Description:')
```

Comment:
Instead bunch of console.log, maybe is a good option to use a template string here.